src/component/selection/setDraftEditorSelection.js

```diff
@@ -303,9 +303,14 @@ function addFocusToSelection(
     // Additionally, clone the selection range. IE11 throws an
     // InvalidStateError when attempting to access selection properties
     // after the range is detached.
-    var range = selection.getRangeAt(0);
-    range.setEnd(node, offset);
-    selection.addRange(range.cloneRange());
```

Comment:
mind adding a failing test for the above and then adding your test which fixes it? That'd prevent a regression